buildchain/buildchain/docker_command.py

```diff
@@ -77,6 +77,25 @@ def decorated_task(*args: Any, **kwargs: Any) -> Optional[TaskError]:
     return wrapped_task
 
 
+class DockerRMI:
+    """
+    A class to expose the `docker rmi` command through the API client.
+
+    This command has `--no-prune` set to True in order to avoid cleaning up
+    intermediary layers that speed up container builds (most notably the GUI).
+    """
+    def __init__(self, name: str, version: str):
```

Comment:
Could take a `ContainerImage` object instead of two separate string.

I mean, it's called `DockerRemoveImage` so I wouldn't be surprised to give an `Image` instance as parameter.

Reply:
This is complicated in the current order of things, as we don't want to depend on `image` for things in `docker_command` or we get hit with circular imports when we type them.

Reply:
The thing is, we already do: you already have an `from buildchain.targets import image` in this file so I don't see where the problem is.